test/integration/integration_test.go

```diff
@@ -394,6 +429,22 @@ func variableShouldNotBeEmpty(variableName string) error {
 	return compareExpectedWithActualNotEquals("", minishift.GetVariableByName(variableName).Value)
 }
 
+func commandReturnShouldMatch(commandField string, expected string) error {
+	return compareExpectedWithActualMatches(expected, selectFieldFromLastOutput(commandField))
+}
+
+func commandReturnShouldNotMatch(commandField string, notexpected string) error {
+	return compareExpectedWithActualNotMatches(notexpected, selectFieldFromLastOutput(commandField))
+}
+
+func commandReturnShouldMatchContent(commandField string, expected *gherkin.DocString) error {
+	return compareExpectedWithActualMatches(expected.Content, selectFieldFromLastOutput(commandField))
```

Comment:
Is the content version using regexp confirmed to work? Does creating a regular expression across multiple new lines work? Aka, does something like this work:

     Then stdout should contain
     """
     - anyuid         :  .*   P(0)
     - admin-user     :  .*  P(0)
     """

In some languages this would not work out of the box. Not sure atm how Go's regexp support handles this.